src/ucp/rma/get_am.c

```diff
@@ -100,6 +100,14 @@ ucp_proto_get_am_bcopy_init(const ucp_proto_init_params_t *init_params)
     return ucp_proto_single_init(&params);
 }
 
+static void ucp_proto_get_am_bcopy_reset(ucp_request_t *request)
+{
+    if (request->flags & UCP_REQUEST_FLAG_PROTO_INITIALIZED) {
```

Comment:
why do we check UCP_REQUEST_FLAG_PROTO_INITIALIZED here, but not in ucp_proto_request_sync_[b/z]copy_reset?

Reply:
probably because setting the flag was missed in `ucp_proto_eager_sync_tag_offload_bcopy_progress`, now it's aligned